src/pages/App.jsx

```diff
@@ -21,6 +22,7 @@ const App = () => (
       <Route path="/forgot-password" component={ForgotPassword}></Route>
       <Route path='/handlesocialauth' component={HandleSocialAuth} />
       <Route path="/feeds" component={connectedFeed} />
+      <Route path="/create-article" exact component={CreateArticle}></Route>
```

Comment:
This is a trivial suggestion. You don't have to implement it. Maybe a route name like /article/create or /articles/create might be better here